webdriver/tests/element_click/link.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click Link Element Tests
+
+def test_numbers_link(session):
+    url = "/webdriver/tests/element_click/support/input.html"
+    session.url = inline("<a href=%s>123456</a>" % url)
+    element = session.find.css("a", all=False)
+    response = click(session, element)
+    assert_success(response)
+
+    assert session.url == "http://web-platform.test:8000%s" % url
```

Comment:
The CI is complaining about literal use of this URL. I believe you can resolve this by relying on the [`server_config`](https://searchfox.org/mozilla-central/source/testing/web-platform/tests/webdriver/tests/support/fixtures.py#127) fixture instead?

Reply:
I think the `url` fixture is what we should use here.

Reply:
So I tried using the url fixture but it kept giving me an error. I did temp = url(server_config()). I imported the function too. from tests.support.fixtures import server_config, url.

Reply:
Mind adding a gist or pastebin with the failure where we could have a look at?